README.md

```diff
@@ -220,6 +220,13 @@ You can set a custom icon by setting the ``slider_icon``attribute to a drawable
 app:slider_icon="@drawable/custom_icon"
 ```
 
+You can also set a custom icon programmtically with the provided setter.
+
+```java
+SlideToActView sta = findViewById(R.id.slider);
+sta.setIcon(R.drawable.custom_icon);
```

Comment:
```suggestion
sta.setSliderIcon(R.drawable.custom_icon);
```

Reply:
Makes sense. Have included in the latest commit.